### PR TITLE
Multibanco payment icon addition

### PR DESCRIPTION
**Pull request by nehagarg-shopify**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/) - I have actually reused the icon for mb. Since there was no way to reuse it as alias, I had to create a new entry for Multibanco
- [x] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.
- [x] I recognize that if my icon is not approved by the Shopify Partners team it may not receive review nor merger.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

File: app/assets/images/payment_icons/multibanco.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....019.01l-.013.006a.228.228 0 01-.17-.009l-.019-.011a.157.157 0 01-.032-.024c-.007-.006-.013-.014-.021-.02l-.004-.007-.839-1.075zM24.298 20.23a.159.159 0 010 .317h-.464a.622.622 0 01-.622-.621v-.475a.625.625 0 01.622-.622h.464c.122 0 .198.132.137.238a.163.163 0 01-.137.078h-.464a.308.308 0 00-.218.093.298.298 0 00-.093.216v.474c0 .17.138.308.308.308h.465zM25.211 19.142a.432.432 0 00-.299.118.394.394 0 00-.121.28v.294a.39.39 0 00.12.279.434.434 0 00.299.118.432.432 0 00.299-.118.389.389 0 00.122-.279v-.294a.394.394 0 00-.122-.28.437.437 0 00-.299-.118m0-.315a.742.742 0 01.517.206.701.701 0 01.217.506v.294a.696.696 0 01-.217.505.743.743 0 01-.517.207.74.74 0 01-.516-.207.702.702 0 01-.218-.505v-.294a.706.706 0 01.218-.506.74.74 0 01.516-.206zM18.565 19.143v1.087h.61a.197.197 0 00.145-.332l-.006-.006a.199.199 0 00-.139-.059h-.21a.158.158 0 01-.137-.236.157.157 0 01.137-.078h.071a.2.2 0 00.119-.057.185.185 0 00-.13-.318h-.46zm-.315.534v-.686c0-.021.003-.042.012-.061a.172.172 0 01.037-.055.154.154 0 01.103-.045h.618c.278 0 .503.226.503.502a.506.506 0 01-.077.268c.033.02.064.043.093.07l.009.012a.51.51 0 01.142.353.515.515 0 01-.513.512h-.77a.157.157 0 01-.157-.158z" fill="#1d1d1b"/></svg>
```
Comment (lunataoo) [inline app/assets/images/payment_icons/multibanco.svg:1]: Can you change the `aria-labelledby="pi-mb"` to `pi-multibanco` as suggested in the [Markup guideline](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md#markup)?

File: db/payment_icons.yml [hunk 2]
```diff
@@ -862,6 +862,10 @@
   name: mb
   label: Multibanco
   group: other
+-
+  name: multibanco
+  label: Multibanco
+  group: other
```
Comment (lunataoo) [inline db/payment_icons.yml:868]: What's happening to the old multibanco icon? Found it [here](https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/mb.svg)

**Review by lunataoo — CHANGES_REQUESTED** [review]

**Comment by rahilvirani** [comment]
@nehagarg-shopify it looks like that you are making changes to Shopify to use the existing multibanco logo. If so, can you please close this PR

**Comment by nehagarg-shopify** [comment]
Closing this PR as we are now passing `mb` as the input to this gem for Multibanco.

**Pull request closed without merging**